invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/GroupTabPane.js

```diff
@@ -80,6 +109,10 @@ export class GroupTabPane extends Component {
                 handleChange={this.addMemberToSelected}
                 searchType="group"
                 placeholder={i18next.t("Search for groups")}
+                existingEntities={existingEntitiesIds}
+                existingEntitiesDescription={i18next.t(
+                  "This group is a member or has already been invited to this community."
```

Comment:
Groups are not invited but simply added:
```suggestion
                  "Already a member"
```